Userland/Applications/PixelPaint/ImageEditor.h

```diff
@@ -45,6 +45,7 @@ class ImageEditor final
     {
         m_guides.remove_first_matching([&](auto& entry) { return &guide == entry.ptr(); });
     }
+    void remove_all_guides() { m_guides.clear(); }
```

Comment:
Please call this `clear_guides()` (or re-title the UI action "Remove All Actions") :^)

Reply:
Ah, I had both while building this. Somehow the mix ended up in the pr.